_posts/2017-03-20-lightning-q1-2017.md

```diff
@@ -0,0 +1,96 @@
+summary: What we've added to our developer toolboxes in Q1 2017!
+---
+
+_This post is part of a new series called "Lightning Talks."_
+
+The web development world evolves quickly, and keeping abreast of new tools and methodologies while simultaneously delivering client work is tough.
+
+One of our core [philosophies](/mission-and-values/#excel) at Savas Labs is to "_hold ourselves to a high and constantly improving standard_." To that end, we dedicate time each month to share new tips and tools with each other to continue increasing our collective knowledge base -- we call these our :zap: "Lightning Skillshares!" :zap:
+
+Since we believe in the open source ethos and sharing with the community, in this series, we'll highlight some tips we use to improve our day-to-day efficiency (and hopefully yours too!).
+
+What we've added to our toolboxes in Q1 2017:
+
+  [1. Using Teleconsole for collaboration](#using-teleconsole-for-collaboration)
+
+  [2. Accessing a Dockerized database via PhpStorm or PhpMyAdmin](#accessing-a-dockerized-database-via-phpmyadmin-or-phpstorm)
+
+  [3. Using Apowersoft's Free Online Screen Recorder to capture your screen](#using-apowersofts-free-screen-recorder-to-capture-your-screen)
+
+## 1. Using Teleconsole for collaboration
+
+[Teleconsole](https://www.teleconsole.com/) let's you share your UNIX terminal session for easy collaboration! We've been using it for a couple months now and find it to be an excellent addition to our team toolbox.
+
+It works by starting an SSH server on the host computer and generating a unique session ID that others can connect to via their terminal or a web browser. You can have multiple connected individuals in a session, though we typically use it more for 1-1 debugging and troubleshooting.
+
+Be aware, collaborators will have _direct_ access to your terminal, so only share with people you trust and be careful about accidental commands. Other than that, it's free and easy to set up -- check it out [here](https://www.teleconsole.com/).
+
+<script type="text/javascript" src="https://asciinema.org/a/7oargr7f911dm8qz4c5m0k2iw.js" id="asciicast-7oargr7f911dm8qz4c5m0k2iw" async></script>
+<span class="caption">Terminal sharing 2,700 miles away!</span>
+
+## 2. Accessing a Dockerized database via PhpMyAdmin or PhpStorm
+
+There are many useful tools available for interacting with a MySQL database. Since we've transitioned to [Docker for our local development](/blog/tag/docker/), we prefer one of the following techniques for manipulating databases abstracted into the Docker container.
+
+### Adding PhpMyAdmin to a Docker Stack
+
+[PhpMyAdmin](https://www.phpmyadmin.net/) is a great tool that provides a useful interface for visualizing and interacting with a MySQL database. [Docker4drupal](https://github.com/wodby/docker4drupal) has created an easy [template](https://github.com/wodby/docker4drupal/blob/master/docker-compose.yml) for adding the PhpMyAdmin service to a project's Docker Stack.
```

Comment:
Hi @lrowang - Docker4Drupal has unfortunately changed their template so can we remove the referenes to Docker4Drupal in this post? The PHPMyAdmin container isn't dependent on Docker4Drupal at all, I just got my inspiration from them! If you're curious, this is the [latest Docker4Drupal template](https://github.com/wodby/docker4drupal/blob/master/docker-compose.yml#L61).